docs/Configuration.md

```diff
@@ -205,22 +205,22 @@ language from being allowed. If there are more than one `locale tag`
 all those must be removed to block that language.
 
 English is the Zonemaster default language, but it can be blocked
-from being allowed by RPC-API by not including it in the
-configuration.
+from being allowed by RPC-API by including some `locale tag` it in the
+configuration, but none starting with language code for English ("en").
```

Comment:
Since we are deprecating the format with the country code, do we need this last part?

Reply:
If LANGUAGE.locale is empty then "en" is supported. There must be no "en_*" and at least some other tag not starting on "en".